data/presets/presets.json

```diff
@@ -1955,6 +1955,51 @@
             },
             "name": "Marketplace"
         },
+        "amenity/monastery": {
+            "icon": "place-of-worship",
+            "fields": [
+                "name",
+                "religion",
+                "denomination",
+                "address",
+                "building_area",
+                "service_times"
```

Comment:
I don't think they do services at monasteries.  Can you remove this field?  Everything else about this looks great 👍